fs/fs_zenfs.cc

```diff
@@ -272,6 +272,11 @@ IOStatus ZenFS::Repair() {
   return IOStatus::OK();
 }
 
+std::string ZenFS::FormatPathLexically(std::string filepath) {
+  std::filesystem::path f = filepath;
+  return f.lexically_normal().string();
```

Comment:
Based on yura's feedback, they expect that "/test.txt" and "test.txt" paths should be equal. So I think the easiest here is to add a prefix slash to the paths, something like:

``` 
 std::string normal = f.lexically_normal().string();
 if (normal.size() == 0 || normal.at(0) != '/') {
   normal = "/" + normal;
 }
 return normal;

Reply:
Ok.

Reply:
@yhr we can unconditionally  add the `/` to the front before calling `lexically_normal()`, then we will always have just 1 leading slash.

Not sure what the check for size is about? Even if path is empty or path is ".", we still want a leading slash, right?

Reply:
Need to check the size before checking index 0 to avoid going out of bounds

Reply:
And we need to check of the slash is present, as we don't want double-slashes if the user has already provided a slash in the file name (lexically_normal does not remove it)

Reply:
See my change suggestion below. Just add the slash before converting to string, then there is no issue. `std::filesystem::path` will remove double slashes.